docs/docs/how-tos/nebari-destroy.md

```diff
@@ -0,0 +1,35 @@
+---
+id: nebari-destroy
+title: Destroy your Nebari deployment
+description: A basic overview to destroying your Nebari deployment and associated resources
+---
+
+## Introduction
+
+This guide is to help first-time users destroy Nebari resources that have been previous deployed while setting up Nebari at the production scale. With your `qhub-config.yaml` configuration file, you can use a handy `destroy` command to automatically destroy resources with Nebari. We will also take a look at how you can manually destroy resources on individual cloud providers in case the automation fails.
```

Comment:
```suggestion
This guide is aimed to guide you through how to tear down Nebari resources previously deployed.

With your `nebari-config.yaml` configuration file, you can use a handy `destroy` command to automatically destroy resources with Nebari. We will also examine how you can manually destroy resources on individual cloud providers in case the automation fails.
```

Reply:
The second sentence is a bit confusing - should be with the yaml file + the nebari CLI no?